src/modality-specific-files/eye-tracking.md

```diff
@@ -0,0 +1,233 @@
+      "CogAtlasID": "RECOMMENDED",
+      "CogPOID": "RECOMMENDED",
+      "DeviceSerialNumber": "RECOMMENDED",
+   }
+) }}
+
+Specific eye-tracking fields MUST be present:
+
+{{ MACROS___make_metadata_table(
+   {
+      "SamplingFrequency__eyetrack": "REQUIRED",
+      "SampleCoordinateUnit": "REQUIRED",
+      "SampleCoordinateSystem": "REQUIRED",
+      "EnvironmentCoordinates": "REQUIRED",
+      "ScreenSize": "REQUIRED",
+      "ScreenResolution": "REQUIRED",
+      "ScreenDistance": (
+         "REQUIRED",
+         "Screen distance in cm (for example 60 for a screen distance of 60 cm), "
+         'if no screen use "n/a". '
+         "For MRI, it corresponds to the distance between the head-coil mirror "
+         "to the projection screen for example."),
+   }
+) }}
+
+Specific eye-tracking fields SHOULD be present:
+
+{{ MACROS___make_metadata_table(
+   {
+      "IncludedEyeMovementEvents": "RECOMMENDED",
+      "DetectionAlgorithm": "RECOMMENDED",
+      "DetectionAlgorithmSettings": "RECOMMENDED",
+      "StartMessage": "RECOMMENDED",
+      "EndMessage": "RECOMMENDED",
+      "KeyPressMessage": "RECOMMENDED",
+      "CalibrationType": "RECOMMENDED",
+      "CalibrationUnit": "RECOMMENDED",
+      "CalibrationPosition": "RECOMMENDED",
+      "MaximalCalibrationError": "RECOMMENDED",
+      "AverageCalibrationError": "RECOMMENDED",
+      "CalibrationList": "RECOMMENDED",
+      "RecordedEye": "RECOMMENDED",
+      "EyeCameraSettings": "RECOMMENDED",
+      "FeatureDetectionSettings": "RECOMMENDED",
+      "GazeMappingSettings": "RECOMMENDED",
+      "RawDataFilters": "RECOMMENDED",
+      "ScreenRefreshRate": "RECOMMENDED",
+      "AOIDefinition": "RECOMMENDED",
+      "PupilFitMethod": "RECOMMENDED",
+   }
+) }}
+
+### Example
+
+```JSON
+{"TaskName": "PSC_train",
+"InstitutionName": "Goethe-University of Frankfurt; Department of Psychology",
+"InstitutionAdress": "Theodor-W.-Adorno-Platz 6 60323 Frankfurt am Main; Germany",
+"Manufacturer": "SR-Research",
+"ManufacturersModelName": "EYELINK II CL v4.56 Aug 18 2010",
+"SoftwareVersion": "SREB1.10.1630 WIN32 LID:F2AE011 Mod:2017.04.21 15:19 CEST",
+"TaskDescription": "Sentence reading",
+"Instructions": "Read sentences as you would read a book or a newspaper",
+"SamplingFrequency": 1000,
+"SampleCoordinateUnit": "pixel",
+"SampleCoordinateSystem": "gaze-on-screen",
+"EnvironmentCoordinates": "top-left",
+"IncludedEyeMovementEvents": [["Start of fixation", "SFIX"],["End of fixation", "EFIX"],["Start of saccade", "SSACC"], ["End of saccade", "ESACC"],["Start of blink", "SBLINK"], ["End of blink", "EBLINK"]],
+"DetectionAlgorithm": "SR-Research",
+"StartMessage": "SENTENCESTART",
+"EndMessage": "SENTENCESTOP",
+"KeyPressMessage": "ANSWER",
+"RecordedEye": "Both",
+"ScreenSize": [38.6, 29],
+"ScreenResolution": [1024, 768],
+"ScreenDistance": 60,
+"AOIDefinition": ["square",["x_start", "x_stop", "y_start", "y_stop"]] ,
+}
+```
+
+### Example Dataset
```

Comment:
```suggestion
### Example Datasets
```

and in general I love when there are nice examples etc. But I feel that it might be a bit too much details per each one, which is more pertinent to be in those datasets instead of here. If you check other `Example datasets` mentioning in other sections, you will see that they are much more concise. 

May be it should be just a list with very brief description and references such as 

### Example Datasets

- Potsdam Sentence Corpus Dataset: Eye-tracking data, measured with an Eyelink, from 2 German speaker learners reading 36 sentences from the Potsdamer Sentence Corpus ([Kliegl et al., 2004](https://doi.org/10.1080/09541440340000213)). [BIDS dataset](https://github.com/greckla/Eye-Tracking-BIDS/tree/master/PSC_train/PSC_train_raw_data_BIDS), [Conversion script](https://github.com/greckla/Eye-Tracking-BIDS/blob/master/PSC_train/from_asc_to_BIDS_asc.Rmd)
- ...

?

Reply:
Yup I made a comment mentioning that those should be links to what examples will go in the bids examples.
https://github.com/bids-standard/bids-specification/pull/1128#discussion_r944255054

Reply:
Ok, good point.